### DISCUSSION TITLE
How to Create the Nested drag and drop using Ignite UI Drag and Drop

### DISCUSSION THREAD

**Discussion by dharmarajmr** [IgniteUI/igniteui-angular#10447] [Q&A]
I'm facing issue when I have decided to create the nested drag and drop. Those features are available in drag n drop ignite UI component.

This is the below sample that you provided in stackblitz 

> https://stackblitz.com/github/IgniteUI/igniteui-live-editing-samples/tree/master/angular-demos/interactions/kanban-sample?file=src%2Fapp%2Fdrag-drop%2Fkanban-sample%2Fkanban-sample.component.ts

but I have to drag the ToDo and Progress and Done article as well. What Should I do?, Thanks

**Comment by zdrawku** [comment]
Hey @dharmarajmr, could you please provide a more detailed explanation on what exactly is the problem that you are facing?
